test/smart/dune

```diff
@@ -16,9 +16,7 @@
 
 (rule
  (alias runtest)
- (package git-unix)
- (enabled_if
-  (= %{os_type} "Unix"))
+ (package git)
```

Comment:
Currently these tests can not work on Windows due to named pipes. We don't have any equivalent with that so, to avoid an error on AppVeyor we execute them only for `unix`.